### PR TITLE
Studio tab nav button styling

### PR DESCRIPTION
**Pull request by fsih**
- Increase the padding in and around tab nav buttons
- Change the color of the hover states

### CODE REVIEW

File: src/views/studio/studio.scss [hunk 1]
```diff
@@ -166,7 +166,7 @@ $radius: 8px;
         border-radius: 8px;
         background: white;
         box-sizing: border-box;
-        border: 2px solid rgba(0, 0, 0, 0.15);
+        border: 2px solid box-shadow-light-gray;
```
Comment (ericrosenbaum) [inline src/views/studio/studio.scss:169]: looks like a typo here

**Review by ericrosenbaum — CHANGES_REQUESTED** [review]
LGTM aside from the one type

**Comment by ericrosenbaum** [comment]
I'm wondering about the "add to studio" modal, which also uses nav buttons. I assume those should be updated to match this change?

File: src/views/studio/studio.scss (hunk 1 above)
Comment (fsih) [reply src/views/studio/studio.scss:169]: Oops thank you!

**Pull request merged**